app/controllers/application_controller.rb

```diff
@@ -24,4 +24,8 @@ def user_not_authorized
   def record_not_found
     render 'errors/not_found'
   end
+
+  def current_company
+    @current_company = current_user.company if current_user
```

Comment:
Перевірку на `current_user` робити не треба, бо зараз, якщо користува незалогінений, у нас завжди буде `Guest`. Лишилось до нього додати пустий метод `#company`.
```suggestion
    @current_company ||= current_user.company
```

Подивись, що `||=` робить.